<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorTTLDialog</class>
 <widget class="QDialog" name="ColorTTLDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>433</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>426</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Color TTL Events</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="C0GB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Green</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QComboBox" name="stream0CB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentText">
         <string>mmmm88</string>
        </property>
        <item>
         <property name="text">
          <string>mmmm88</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="analog0CB">
        <item>
         <property name="text">
          <string>Digital bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Analog channel</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="chan0SB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="thresh0Lbl">
        <property name="text">
         <string>Analog thresh volts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="T0SB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-5.000000000000000</double>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="C1GB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Magenta</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QComboBox" name="stream1CB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>mmmm88</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="analog1CB">
        <item>
         <property name="text">
          <string>Digital bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Analog channel</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="chan1SB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="thresh1Lbl">
        <property name="text">
         <string>Analog thresh volts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="T1SB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-5.000000000000000</double>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="C2GB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Cyan</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QComboBox" name="stream2CB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>mmmm88</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="analog2CB">
        <item>
         <property name="text">
          <string>Digital bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Analog channel</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="chan2SB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="thresh2Lbl">
        <property name="text">
         <string>Analog thresh volts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="T2SB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-5.000000000000000</double>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="C3GB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Orange</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QComboBox" name="stream3CB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>mmmm88</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="analog3CB">
        <item>
         <property name="text">
          <string>Digital bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Analog channel</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="chan3SB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="thresh3Lbl">
        <property name="text">
         <string>Analog thresh volts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="T3SB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-5.000000000000000</double>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="widthSB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Stay high count</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Minimum width (s)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="inarowSB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="helpBut">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>C0GB</tabstop>
  <tabstop>stream0CB</tabstop>
  <tabstop>analog0CB</tabstop>
  <tabstop>chan0SB</tabstop>
  <tabstop>T0SB</tabstop>
  <tabstop>C1GB</tabstop>
  <tabstop>stream1CB</tabstop>
  <tabstop>analog1CB</tabstop>
  <tabstop>chan1SB</tabstop>
  <tabstop>T1SB</tabstop>
  <tabstop>C2GB</tabstop>
  <tabstop>stream2CB</tabstop>
  <tabstop>analog2CB</tabstop>
  <tabstop>chan2SB</tabstop>
  <tabstop>T2SB</tabstop>
  <tabstop>C3GB</tabstop>
  <tabstop>stream3CB</tabstop>
  <tabstop>analog3CB</tabstop>
  <tabstop>chan3SB</tabstop>
  <tabstop>T3SB</tabstop>
  <tabstop>inarowSB</tabstop>
  <tabstop>widthSB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorTTLDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
